api/models/app.go

```diff
@@ -59,7 +59,32 @@ func ListApps() (Apps, error) {
 }
 
 func GetApp(name string) (*App, error) {
-	res, err := DescribeStack(name)
+	return GetAppFast(name, true, true)
+}
+
+func GetAppFast(name string, bound bool, unbound bool) (*App, error) {
```

Comment:
I'm having a tough time with this function definition. My first impression is that it's hard to know how to use it and what to expect.